src/Sulu/Component/Webspace/Manager/WebspaceManager.php

```diff
@@ -319,7 +319,9 @@ public function getWebspaceCollection()
                 );
             }
 
-            require_once $cache->getPath();
+            if (!class_exists($class)) {
```

Comment:
Why is that suddenly necessary? Feels a bit like hiding a bug...

Reply:
when I did debug it. It seems like the $container from the `TestCase` called require_once and the container from `$client` did call require_once as they have different WebspaceManager instances. It did end up with: `Cannot declare class Test, because the name is already in use`